Makefile

```diff
@@ -30,7 +33,38 @@ config:
 	$(BUILD_ENV_ARGS) \
 		$(MAKE) -C buildroot O=output piksiv3_defconfig
 
+dev-tools-clean: pkg-piksi_dev_tools-dirclean
+dev-tools-build: pkg-piksi_dev_tools
+
+rel-lockdown-clean: pkg-release_lockdown-dirclean
+
+image-release-open: export BR2_BUILD_RELEASE_OPEN=y
+image-release-open: config
+	$(BUILD_ENV_ARGS) \
+		$(MAKE) flush-rootfs rel-lockdown-clean
+	$(BUILD_ENV_ARGS) \
+		$(MAKE) -C buildroot O=output V=$(V)
+
+image-release-protected: export BR2_BUILD_RELEASE_PROTECTED=y
+image-release-protected: config
+	$(BUILD_ENV_ARGS) \
+		$(MAKE) flush-rootfs rel-lockdown-clean
+	[ -z "$BR2_BUILD_PIKSI_INS" ] || \
```

Comment:
I needed to change this to `"$(BR2_BUILD_PIKSI_INS)"` to get this to work.

Reply:
resolved